language/tools/move-mv-llvm-compiler/tests/move-ir-tests/struct-cmp.move

```diff
@@ -0,0 +1,39 @@
+module 0x100::cmp_struct_tests {
```

Comment:
This test case produces voluminous output and can be entirely replaced with a much simpler one as shown below. It is desirable in the `move-ir-tests` to keep the IR output as small as possible, since these have to be remastered and eyeball-checked when the IR output changes for any reason.
```
module 0x100::foo {
    fun doit(a: vector<u64>, b: vector<u64>): bool {
        a != b
    }
}
```